### ISSUE TITLE
Steam freezing the entire computer after startup.

### ISSUE THREAD

**Issue by CaioFrancisco** [ValveSoftware/steam-for-linux#7683]
#### Your system information

* Steam client version (build number or date): February 12 2021
* Distribution (e.g. Ubuntu): Linux Mint (20.1)
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes
 
#### Please describe your issue in as much detail as possible:
Under certain conditions, Steam after logging into my account, freezes the entire computer. The mouse gets frozen, the screen gets frozen, all applications are extremely slow, if not outright frozen too. And the Steam window never appears. Until i close all Steam instances, theses symptoms won't disappear, until, of course, Steam gives a segmentation fault (if requested, I'll provide the multiple core dumps.)

#### Steps for reproducing this issue:

(Theses are the steps to fix a V-Sync issue in all Intel Graphics drivers.)

1. Create a folder under /etc/X11/ called "xorg.conf.d"
2. Inside this folder, create a file called "20-intel.conf"
3. Inside this file, insert [what's written here.](https://gist.github.com/CaioFrancisco/dfd34925c82b121105548397135122e4)
4. Reboot your machine.

**Comment by kisak-valve** [comment]
Hello @CaioFrancisco, no OpenGL application including Steam should be capable of freezing the entire system. This reads like a video driver issue and should be reported to your video driver vendor if it hasn't been already.

**Comment by CaioFrancisco** [comment]
I figured out this solution has a drawback: it requires more memory allocation and performance. And that might've been what ended up crashing my whole system. Thank you for answering though.

**Comment by CaioFrancisco** [comment]
Sorry for re-opening the issue after 1 month, but I've discovered a "patch" to the issue. It turns out that the little patch I used overwrote the "Device" section _completely_ on the default Xorg configuration. This means it sets the "DRI" back to whatever it was before it was set to "3". The main issue is that Vulkan relies on the DRI 3, and since DRI isn't 3, it doesn't work correctly. Although this was an user error, I would like a small patch that just makes a check to see if the DRI is 3 so it doesn't eat the RAM away until Steam gives a segmentation fault. And yes, turning DRI to 3 solved the issue.
